include/net/wifi.h

```diff
@@ -19,6 +19,7 @@ enum wifi_security_type {
 
 #define WIFI_SSID_MAX_LEN 32
 #define WIFI_PSK_MAX_LEN 64
+#define WIFI_MAC_MAX_LEN 64
```

Comment:
Is MAC actually that long? Isn't it just 6 bytes?

Reply:
Yes, it is 6 bytes. I will fix. Thanks for noticing this copy-paste leftover here. MAC address is stored as a string like format, so this is supposed to be a mac address length as a string "11:22:33:44:55:66" -> 17.

Reply:
I'd rather store 6 bytes only and format into string whenever showing it to the user.

Reply:
Yeah, agree that it would be better. I took different approach for that as it was straight-forward for this use case in this PR, i.e. show mac addresses in "wifi scan", but I'll check this more.

Reply:
I agree we should rather store the binary form, there's a `net_sprint_ll_addr()` function already available, could be useful for wifi shell.

Reply:
Done.